src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -119,7 +119,7 @@ public static void MapKeyboard(ISearchBarHandler handler, ISearchBar searchBar)
 			handler.PlatformView?.UpdateKeyboard(searchBar);
 		}
 
-		static void MapFocus(ISearchBarHandler handler, ISearchBar searchBar, object? args)
+		public static void MapFocus(ISearchBarHandler handler, ISearchBar searchBar, object? args)
```

Comment:
we can probably leave this private?

Reply:
We can but the rest are public and I was going to make it public for .NET 8 - unless you see a reason otherwise?